CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+Contributor License
+-------------------
+
+VOLTTRON is an Eclipse Foundation project.  If this is your first time contributing to an Eclipse Foundation project, you'll need to sign the [Eclipse Contributor Agreement][ECA].
+
+- [Create an account](https://dev.eclipse.org/site_login/createaccount.php) on dev.eclipse.org
+- Open your [Account Settings tab](https://dev.eclipse.org/site_login/myaccount.php#open_tab_accountsettings), enter your GitHub ID and click Update Account
+- Read and [sign the ECA](https://dev.eclipse.org/site_login/myaccount.php#open_tab_cla)
+- Use the exact same email address for your Eclipse account and your commit author.
+
+Issues
+------
+
+Search the [issue tracker](https://github.com/volttron/volttron-core/issues) for a relevant issue or create a new one.
+
+Making changes
+--------------
+
+Fork the repository in GitHub and make changes in your fork.
+
+Please add a description of your changes to the [draft release notes](RELEASE_NOTE_DRAFT.md).
+
+Finally, submit a pull request.
+
+Contact us
+----------
+
+[Join the mailing list][mailing-list] and email the community at volttron@pnnl.gov to discuss your ideas and get help.

```

Comment:
Is there supposed to be a hyperlink here? How does this render on a webpage? This is what I see: 
![Screen Shot 2022-07-18 at 2 17 09 PM](https://user-images.githubusercontent.com/6901848/179619030-f4aa862c-4d7d-456f-bf55-79109fb6b6a0.png)